### PR TITLE
Replace `--network-type` with `--planet`

### PR DESCRIPTION
**Pull request by longfin**
As title says, this PR replace `--network-type` argument with `--planet` to specify planet which node will be belongs to.

### CODE REVIEW

File: NineChronicles.Headless.Executable/Program.cs [hunk 1]
```diff
@@ -250,6 +253,11 @@ await CoconaLiteApp.CreateHostBuilder()
             var headlessConfig = new Configuration();
             configuration.Bind("Headless", headlessConfig);
 
+            if (networkType is { })
+            {
+                Log.Warning("networkType has been deprecated and has no effect. please use `planet` instead.");
```
Comment (moreal) [inline NineChronicles.Headless.Executable/Program.cs:258]:
```suggestion
                Log.Warning("'--network-type' option has been deprecated and has no effect. please use `--planet` instead.");
```

File: NineChronicles.Headless/NineChroniclesNodeService.cs [hunk 2]
```diff
@@ -255,23 +257,20 @@ StandaloneContext context
...
-                NetworkType.Internal => source.GetInternalPolicy(),
-                NetworkType.Permanent => source.GetPermanentPolicy(),
-                NetworkType.Test => source.GetTestPolicy(),
-                NetworkType.Default => source.GetDefaultPolicy(),
-                _ => throw new ArgumentOutOfRangeException(nameof(networkType), networkType, null),
+                Planet.Odin => source.GetPolicy(),
+                Planet.OdinInternal => source.GetInternalPolicy(),
+                _ => throw new ArgumentException(
+                    $"Can't retrieve policy for given planet ({planet})",
+                    nameof(planet)
```
Comment (moreal) [inline NineChronicles.Headless/NineChroniclesNodeService.cs:269]: Is it okay for the heimdall network case?

**Review by moreal — COMMENTED** [review]
👍🏻

File: NineChronicles.Headless/NineChroniclesNodeService.cs (hunk 2 above)
Comment (longfin) [reply NineChronicles.Headless/NineChroniclesNodeService.cs:268]: ⌛

Comment (longfin) [reply NineChronicles.Headless/NineChroniclesNodeService.cs]: @moreal I moved this dispatching to Lib9c side. PTAL https://github.com/planetarium/lib9c/pull/2326 🙏

**Review by moreal — APPROVED** [review]

**Pull request merged**